### DISCUSSION TITLE
Multiple "hosts" created?

### DISCUSSION THREAD

**Discussion by Sazerak** [Security-Onion-Solutions/securityonion#2400] [Unsupported Versions] [answered]
I was just watching Security Onion Essentials - Intro to Analyst Tools on YouTube and the person on the video used the command so-test to test downloading PCAPs, or something to that effect. Anyways, I did it on mine setup, and all of a sudden, I looked at my Unifi dashboard, and I had about 800 new clients (hosts). While I can't say that this was the cause, it's hard to imagine it's a coincidence.

The hosts only stayed up for about 7 minutes, so I'm sure it's nothing concerning, but it sure took me by surprise. Is it normal for the test to create all these connections?

**Comment by dougburks** [comment, marked as answer]
so-test replays pcap samples to your sniffing interface. If your sniffing interface is allowed to send traffic back to your network, then it's possible that your Unifi dashboard was showing the hosts represented in those pcap samples.

**Reply by Sazerak** [reply]
Good to know, thanks. Would it be advisable then to only allow traffic to flow one-way for the sniffer? Or could that have an impact on other features?

**Reply by dougburks** [reply]
Taps and span ports are usually configured to only send traffic to the sniffing interface and not allow traffic from the sniffing interface back to the rest of the network.
